src/ansys/geometry/core/designer/face.py

```diff
@@ -308,9 +341,57 @@ def __grpc_edges_to_edges(self, edges_grpc: List[GRPCEdge]) -> List[Edge]:
         List[Edge]
             ``Edge`` objects to obtain from gRPC messages.
         """
+        from ansys.geometry.core.designer.edge import CurveType, Edge
+
         edges = []
         for edge_grpc in edges_grpc:
             edges.append(
-                Edge(edge_grpc.id, CurveType(edge_grpc.curve_type), self._body, self._grpc_client)
+                Edge(
+                    edge_grpc.id,
+                    CurveType(edge_grpc.curve_type),
+                    self._body,
+                    self._grpc_client,
+                    edge_grpc.is_reversed,
+                )
             )
         return edges
+
+    def create_isoparametric_curves(
+        self, use_u_param: bool, parameter: float
+    ) -> List[TrimmedCurve]:
+        """
+        Create isoparametic curves at the given proportional parameter.
+
+        Typically, only one curve will be created, but if the face has a hole, it is possible to
+        create more than one curve.
+
+        Parameters
+        ----------
+        use_u_param : bool
+            Define whether the parameter is the u coordinate or v coordinate. If True,
+            then u parameter is used. If False, then v parameter is used.
+        parameter : float
+            The proportional [0-1] parameter to create the curvse at.
```

Comment:
```suggestion
            Proportional [0-1] parameter to create the one or more curves at.
```
Does the "[0-1]" indicate a range of values? If so, I think rewriting to this would be clearer: Proportional parameter to create the one or more curves at. The value for this parameter must be from 0 to 1 inclusive.